### ISSUE TITLE
Home Pages - Project page display

### ISSUE THREAD

**Issue by tagapic** [geotagx/geotagx-theme#112]
@elliec 
The Projects page – Used the Chrome as browser. May I suggest some amendment to the look. It now shows 4 featured projects at the top and there is no button to see “all projects”. I juggle around to see the full list (actually search for the incomplete projects) or simply use my page history to go back to the subportal that I wish to work on.

Would also suggest auto arrangement by "unfinished projects" and "completed projects (100%)". The “order by progress” would be thus eliminated as the volunteer is primarily interested in working on unfinished projects and can go and browse the finished one if s/he likes. The "order by progress" menu currently does not seem to order the projects in terms of % completed.

On the same page, it would be useful to have an explanation of "order by activity" and "order by progress". The logic of display of the projects (line-up) for "order by activity" is also not clear. Assume "order by activity" relates to "tasks" and if so the title <b>could</b> be "order by tasks" or "order by tasks completed" (based on what it is supposed to be displaying). 

Best,

**Comment by elliec** [comment]
@tagapic good idea for ordering by unfinished and completed. This is definitely something we could implement, i will create an issue for it.

I am a bit confused by the "order by progress", "order by activity", and "order by tasks" menu that you are talking about though. Could you send me the link to the page you are talking about?

When I look at the projects page http://geotagx.org/#project_grid I see the 7 categories, the first one of which is "all" and will show all of the projects (though in random order)

**Comment by elliec** [comment]
Action: On the http://geotagx.org/#project_grid, projects should be ordered by completeness, with those that are 100% at the bottom of the list.

**Comment by tagapic** [comment]
Thanks @elliec 
Here is how I came upon the Oder by Name, Progress, etc...
From <a href="http://geotagx.org/about">About Page</a> - the user is given some guidance as below.

<b><center>How to get started? +</b>
<b>How will the analysis help? -</b></center>. <i>When you open the + sign, you see text as below...</i>
Let’s see with an example.
Our <a href="http://geotagx.org/project/category/emergencyshelterassessmentinthemiddleeast/">Winter Shelter Assessment application</a> is designed to assess whether emergency shelters in displacements sites in Jordan and Iraq are suitably prepared for winter.

![screenshot tagproject-x](https://cloud.githubusercontent.com/assets/14006578/10295759/eb441d48-6b90-11e5-9191-a15d5a2811d0.png)

From the snapshot you will see the options for Ordering by name, date, activity and progress given on the page, and referred to in my comment. 

As mentioned earlier, it was not clear to me what user is expected to see under "order by activity" and "order by progress". The logic of display of the projects (line-up) for "order by activity" is also not clear. I assumed "order by activity" related to "tasks" and if so the title could be "order by tasks" or "order by tasks completed" (based on what it is supposed to be displaying).  This order by "tasks" makes sense as it is a unit of display, the other unit is the image Number, as for each task there are several images....

Hope the problem is clearer... thanks,

**Comment by elliec** [comment]
@tagapic I see why this is confusing, @spMohanty Do you think we could hide this option for now in the categories because none of them have so many projects that there is a need to order them.
